circuit_maintenance_parser/output.py

```diff
@@ -49,7 +49,7 @@ class Status(str, Enum):
     RE_SCHEDULED = "RE-SCHEDULED"
 
 
-class CircuitImpact(BaseModel):
+class CircuitImpact(BaseModel, extra=Extra.forbid):
```

Comment:
Wow, I didn't know about this syntax option for specifying model configuration - that's pretty nifty!